### PR TITLE
Simple mob alien code cleanup + Couple nerfs to the queen

### PR DESCRIPTION
**Pull request by ghost**
Thanks for pointing out the issues, stealth. Queen 1v1ing a collosus shouldn't be possible.

Changes:

- Queen neuro is no longer hilariously deadly

- Queen no longer rapidfires, same with sentinel, whoops

- Queen is now properly slow

- Queen now summons backup when first spotting you

Code changes:

Fixes up shitcode, adds in proper var use, gets rid of a couple useless lines, fixes alien queens shrieking and spinning like tops nonstop when not supposed to

##### Changelog

:cl:
rscadd: Alien empress simplemobs have been made slightly less hilariously overpowered.
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
whoops

**Comment by ShadowDeath6** [comment]
whoops.dm

**Comment by BOBsHERE** [comment]
are all xenos still op? when a normal xeno mob is nearly dead it does the rapid fire hit crit stun. while they cant kill megafauna i have seen them take them down to half health

**Comment by Super3222** [comment]
I did warn you.

File: code/modules/mob/living/simple_animal/hostile/alien.dm [hunk 1]
```diff
@@ -215,12 +215,12 @@
 /mob/living/simple_animal/hostile/alien/queen/large/adjustHealth(amount)
 	if(angry == 0)
 		if((health < maxHealth/2) && if(stat != DEAD))
```
Comment (X-TheDark) [inline code/modules/mob/living/simple_animal/hostile/alien.dm]: `if()` inside another `if()`-------------------------------^^^^^^^^^^^.

This is the actual problem that's causing issues...

**Pull request merged**